app/assets/javascripts/ui_models/application_group.ts

```diff
@@ -121,6 +123,10 @@ export class ApplicationGroup {
     if (this.application) {
       callback();
     }
+
+    return () => {
+      pull(this.changeObservers, callback)
```

Comment:
Looking at the lodash source, using `pull` seems overkill when we only need to check for referential equality. I think a combination of `indexOf` and `splice` would be more efficient, maybe as a utility function? (see `utils.ts`)

Reply:
Updated.

Reply:
SNJS has slowly been in the process of removing dependence on lodash in favor of utils. So we have [`removeFromArray`](https://github.com/standardnotes/snjs/blob/004/lib/utils.ts#L175) that can be imported from SNJS.

Reply:
Thanks for the info. I will remove lodash in mobile code completely in 004.

Reply:
We don't have to totally phase out lodash if it's difficult. Just sort of a gradual process is good enough. But in most cases I believe snjs's utils collection offers a good replacement for most of the basic lodash utils.